codesyncapp/codesync-vs
# Fix glob

- Ignorning symlinks and other types. Only dealing with files in globSync.
- Iterating only 1 repo in populateBuffer in one iteration
- For currently opened repo, waiting for 4 minutes and for other repos, waiting for 2 minutes to run populateBuffer

Testing instructions
- Add some symlink in the repo and make sure there is no code break
- For currently opened repo, populateBuffer should wait for 4 minutes and for other repos it should wait for 2 miuntes
